css/css-flexbox/META.yml

```diff
@@ -1,9 +1,6 @@
 links:
 - product: firefox
-  results:
-  - test: position-absolute-001.html
   url: https://bugzilla.mozilla.org/show_bug.cgi?id=1543267
-- product: firefox
   results:
-  - test: position-absolute-002.html
-  url: https://bugzilla.mozilla.org/show_bug.cgi?id=1543267
\ No newline at end of file
+  - test: position-absolute-001.html
+  - test: position-absolute-002.html
```

Comment:
@AutomatedTester - whoops, missed this in #30. We avoid duplicate urls by allowing multiple results nodes per url.